src/test/resources/features/Functional.feature

```diff
@@ -1,5 +1,6 @@
 Feature: Functional Test
 
```

Comment:
Can you remove the ignore? I want to see that both Functional and Non-Functional Scenario work in the pipeline

Reply:
Sure, later I will remove the tag from the feature file...